packages/launchpad/src/setup/OpenBrowser.vue

```diff
@@ -113,10 +120,12 @@ const launch = async () => {
 }
 
 const launchIfBrowserSetInCli = async () => {
-  const isValidBrowser = query.data.value?.localSettings.preferences.isValidBrowser
-  const wasBrowserSetInCli = query.data.value?.localSettings.preferences.wasBrowserSetInCLI
+  await lsQuery
```

Comment:
I had to await `lsQuery` here to make sure that the data was available without passing as a prop.